include/st_pipeline_api.h

```diff
@@ -550,6 +550,13 @@ enum st20p_rx_flag {
    * Pls note it could fallback to CPU if no DMA device is available.
    */
   ST20P_RX_FLAG_DMA_OFFLOAD = (MTL_BIT32(17)),
+   /**
+   * Flag bit in flags of struct st20p_rx_ops.
+   * Only for ST20P_TYPE_FRAME_LEVEL/ST20P_TYPE_SLICE_LEVEL.
```

Comment:
No ST20P_TYPE_FRAME_LEVEL/ST20P_TYPE_SLICE_LEVEL for st20p mode, can you remove this?

Reply:
Yes, it's removed.